ercc/attestation/Makefile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: all clean
+
+IAS_GO_DEFINES=
+
+ifeq (${SGX_MODE}, HW)
```

Comment:
Can we treat SGX_MODE by default as HW?!

Reply:
Yes. I have added that in the top level config.mk